cluster/utils_windows.go

```diff
@@ -0,0 +1,15 @@
+// +build !linux,!darwin
+
+package cluster
+
+import (
+	"errors"
+	"net"
+	"time"
+)
+
+// SetTCPUserTimeout doesn't support Windows because Go doesn't support
+// the option and swarm doesn't support cgo
+func setTCPUserTimeout(conn *net.TCPConn, uto time.Duration) error {
+	return errors.New("Go doesn't have native support for TCP_USER_TIMEOUT for this platform")
```

Comment:
I'm concerned about this, does this mean that a Manager can't be started on a Windows Host because it does not support `TCP_USER_TIMEOUT`?